tests/devicetestcase.py

```diff
@@ -30,7 +30,8 @@ def setUpClass(cls):
         if os.getenv('TEST_DEVICES'):
             cls.devices = os.getenv('TEST_DEVICES').split()
             for device in cls.devices:
-                util.dd("/dev/zero", device, util.dev_size_mb(device), bs="1M")
+                cls.size_mb = util.dev_size_mb(device)
```

Comment:
As I recall, there were 2 cases. 1st case where `TEST_DEVICES` is not set and the tests use 1 loopback device. And 2nd case with usually 2 devices created or attached to machine in advance. Usually they are of the same size, but shouldn't be an array of sizes by device?

Reply:
Makes sense. Ok.